css/style.css

```diff
@@ -0,0 +1,442 @@
+body {
+	font-family: 'GothamPro';
+	max-width: 1030px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5980773)